.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,92 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["Triage: Needed"]
+body:
+  - type: markdown
+    attributes:
+      value:
+        Thanks for taking the time to fill out this bug report!
```

if you add a text not inline to field add pipe
```suggestion
      value: |
        Thanks for taking the time to fill out this bug report!
```